### PR TITLE
Use managed cluster for ZTS deployment deletion

### PR DESCRIPTION
**Pull request by dhaiducek**
Addresses:
- https://github.com/stolostron/backlog/issues/25429

### CODE REVIEW

**Comment by dhaiducek** [comment]
The checks have passed here, and this PR is ready for review.

File: test/integration/policy_zts_cmc_test.go [hunk 1]
```diff
@@ -111,13 +111,13 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the zts-cmc policy", Ordered,
...
-			"get", "deployment", "-n", deploymentNS, deploymentName, "--kubeconfig="+kubeconfigHub,
+		_, err := utils.KubectlWithOutput(
+			"get", "deployment", "-n", deploymentNS, deploymentName, "--kubeconfig="+kubeconfigManaged,
 		)
-		Expect(out).To(ContainSubstring(deploymentName))
+		Expect(err).To(BeNil())
 
-		_, err := utils.KubectlWithOutput(
-			"delete", "deployment", "-n", deploymentNS, deploymentName, "--kubeconfig="+kubeconfigHub,
+		_, err = utils.KubectlWithOutput(
```
Comment (mprahl) [inline test/integration/policy_zts_cmc_test.go:119]: Should the delete happen after the policy is deleted?

Comment (dhaiducek) [reply test/integration/policy_zts_cmc_test.go:119]: Good point!

Comment (dhaiducek) [reply test/integration/policy_zts_cmc_test.go:120]: I'm also not sure why the `get` was in there, so I removed it.

**Review by mprahl — APPROVED** [review]

**Pull request merged**